src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -173,6 +173,21 @@ public Category GetCategory
         }
 
         /// <summary>
+        /// Get Element Type
+        /// </summary>
+        public Element ElementType
+        {
+            get
+            {
+                // Get the active Document
+                Autodesk.Revit.DB.Document document = DocumentManager.Instance.CurrentDBDocument;
+
+                return document.GetElement(this.InternalElement.GetTypeId()).ToDSType(true);
```

Comment:
you are correct, although the error would be in `ToDSType()` method getting called on `null` value since `GetElement(ElementId.InvalidElementId)` would just return 'null'. Should that method be able to better handle `null` values?